SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/WorkerCoordinator/CoordinatorConnection.cs

```diff
@@ -47,32 +47,20 @@ private static void KeepConnectionAlive(Connection connection, Logger logger)
         {
             var thread = new Thread(() =>
             {
-                using (var dispatcher = new Dispatcher())
-                {
-                    var isConnected = true;
-
-                    dispatcher.OnDisconnect(op =>
-                    {
-                        logger.WriteError("[disconnect] " + op.Reason, logToConnectionIfExists: false);
-                        isConnected = false;
-                    });
+                Dispatcher dispatcher = new Dispatcher();
```

Comment:
I have no idea how C# works, but Dispatcher is no longer IDisposable.

Reply:
Can't find the change in the release note, but if it's not disposable then this seems correct to me.